packages/golden-layout/karma.conf.cjs

```diff
@@ -24,7 +24,14 @@ module.exports = function (config) {
     },
 
     browserify: {
+      debug: true,
       plugin: ['esmify'],
+      // ignored because it doesn't understand the scss import in GoldenLayoutThemeExport
+      configure: function (bundle) {
+        bundle.on('prebundle', function () {
+          bundle.ignore('./dist/GoldenLayoutThemeExport.js');
```

Comment:
Can't you ignore the scss file instead? EDIT: Ah but then the js file would puke. Nevermind